src/ngraph/frontend/onnx_import/ops_bridge.hpp

```diff
@@ -107,6 +112,8 @@ namespace ngraph
                                     const std::string& domain,
                                     Operator fn);
             OperatorSet _get_operator_set(std::int64_t version, const std::string& domain);
+            OperatorSet _get_operator_set(const std::string& domain);
```

Comment:
Redundant definition. Having those two methods introduces confusion and inconsistency about the API. Why single API call with default parameters is not enough?